ts/sagas/onboarding.ts

```diff
@@ -100,6 +104,23 @@ function* tosCheckSaga(): Iterator<Effect> {
     // Here we wait the user accept the ToS
     yield take(TOS_ACCEPT_REQUEST);
 
+    // Get the current user profile from the store
```

Comment:
this whole block should be in a loop until `isInboxEnabled === true` and `isWebhookEnabled === true`

Reply:
Yes error handling, as said in the PR comment, it the WIP part of this PR.